src/Symfony/Component/AssetMapper/ImportMap/RemotePackageStorage.php

```diff
@@ -90,6 +88,18 @@ public function getDownloadPath(string $packageModuleSpecifier, ImportMapType $i
         return $this->vendorDir.'/'.$filename;
     }
 
+    private function saveFile(string $filename, string $contents): void
+    {
+        if (!is_dir($vendorDir = \dirname($filename)) && !@mkdir($vendorDir, 0777, true)) {
+            throw new RuntimeException(sprintf('The directory "%s" does not exist and could not be created.', $vendorDir));
+        }
+        if (!is_writable($vendorDir)) {
+            throw new RuntimeException(sprintf('The directory "%s" is not writable.', $vendorDir));
+        }
+
+        file_put_contents($filename, $contents);
```

Comment:
if (false === etc then throw?

Reply:
If you meant: 
```php
        if (false === file_put_contents($filename, $contents)) {
            throw new RuntimeException(sprintf('The file "%s" could not be written to.', $filename));
        }
```        

Updated, thanks :)